src/app/clusters/ecosystem-information-server/ecosystem-information-server.cpp

```diff
@@ -222,8 +222,9 @@ std::unique_ptr<EcosystemLocationStruct> EcosystemLocationStruct::Builder::Build
         ChipLogError(Zcl, "Location Name must be less than %u bytes", static_cast<uint16_t>(kLocationDescriptorNameMaxSize)));
 
     // std::make_unique does not have access to private constructor we workaround with using new
```

Comment:
This comment is false.

Reply:
https://github.com/project-chip/connectedhomeip/pull/38205